EasyPost/Models/API/PaymentMethod.cs

```diff
@@ -59,23 +31,30 @@ public class PaymentMethodObject : Resource
         [JsonProperty("exp_year")]
         public int exp_year { get; set; }
         // both
-        [JsonProperty("id")]
-        public string id { get; set; }
+
         // both
         [JsonProperty("last4")]
         public string last4 { get; set; }
         // credit_card
         [JsonProperty("name")]
         public string name { get; set; }
         // both
-        [JsonProperty("object")]
-        public string Object { get; set; }
+
```

Comment:
Issue: Appears to be some duplication of extra spaces and comments here. Let's clean this up

Reply:
Resolved